src/Phinx/Db/Table.php

```diff
@@ -188,6 +188,19 @@ public function changePrimaryKey($columns)
         return $this;
     }
 
+    /**
+     * Checks to see if a primary key exists.
+     *
+     * @param string|string[] $columns Column(s)
+     * @param string|null $constraint Constraint names
+     *
+     * @return bool
+     */
+    public function hasPrimaryKey($columns, $constraint = null)
```

Comment:
You could add a type for the 2nd param it seems. Same for return value

Reply:
I would not want to add type hinting to just one function in this file for fear of overall consistency. Perhaps on a release of 0.13, could do a quick release of 0.14 which just adds type hints where possible across the entire codebase, and then going forward type hints are always used?

Could also land such work in 0.13, but I'd almost at this point just say I'll get back to and land #1962 and then release instead of having it drag out too much longer.

Reply:
Consistency sure is a point.
But this is a new public API method that is added.
And I think the benefits of types in place weigh higher than the consistency argument (from user perspective anyway).
I agree that any 0.13 semantic major release would need to clean up all possible types.